libs/providers/flagd-web/src/lib/flagd-web-provider.spec.ts

```diff
@@ -251,19 +275,20 @@ describe(FlagdWebProvider.name, () => {
     beforeEach(() => {
       mockCallbackClient = new MockCallbackClient();
       OpenFeature.setProvider(
+        'shutdown test',
         new FlagdWebProvider(
           { host: 'fake.com', maxRetries: -1 },
           console,
           mockPromiseClient,
-          mockCallbackClient as unknown as CallbackClient<typeof Service>
-        )
+          mockCallbackClient as unknown as CallbackClient<typeof Service>,
+        ),
       );
     });
 
     describe('API close', () => {
-      it('should call cancel function on provider', () => {
+      it('should call cancel function on provider', async () => {
         expect(mockCallbackClient.cancelFunction).not.toHaveBeenCalled();
-        OpenFeature.close();
+        await OpenFeature.close();
```

Comment:
This test was actually broken (missing the await) but it didn't appear to be because of test interference. Adding provider names to every test helps with this sort of thing.